processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -80,265 +64,10 @@
 
 /**
  */
-public abstract class IncrementalIndex<AggregatorType> implements Iterable<Row>, Closeable
+public abstract class IncrementalIndex<AggregatorType> extends IncrementalIndices implements Iterable<Row>, Closeable
```

Comment:
why does this need to "extend" IncrementalIndices , it seems that only contains a bunch of "static" stuff that needs to be referenced by IncrementalIndex.